test/probes/https/client.js

```diff
@@ -1,11 +1,24 @@
+'use strict'
+
 exports.data = function (ctx) {
   return {
     url: 'https://localhost:' + ctx.data.port + '/?foo=bar'
   }
 }
 
 exports.run = function (ctx, done) {
-  ctx.https.get(ctx.data.url, function (res) {
+  let options = {
+    ecdhCurve: 'auto',
+    protocol: 'https:',
+    host: 'localhost',
+    port: ctx.data.port,
+    path: '/?foo=bar'
+  }
+  // following fails on 8.9.0 - must use options above with ecdhCurve specified. That test
+  // fails because the port is missing in the remoteUrl property. Upgraded node to 8.12.0
+  // so simple URL works.
+  options = ctx.data.url
```

Comment:
Good question. I left the redundant code in because, even though I changed the version of node to get the fix for node's bug.

1. it documented how to fix the node bug
2. it was in testing code so wouldn't appear in the distribution nor impact run-time.